#153470982 Add Cost Price measure for a product

tosmak16 opened the pull request:
#### What does this PR do?
It adds cost price field to product variant schema
it adds cost price input field to product variant form
#### Description of Task to be completed?
* Add cost price field to product variant schema
* Add cost price input field to product variant form
#### How should this be manually tested?
* Run the application
* Add a product 
#### Any background context you want to provide?
This feature is a prerequisite to actionable analytics feature
#### What are the relevant pivotal tracker stories?
#153470982
#### Screenshots
![image](https://user-images.githubusercontent.com/30419014/33714303-c02a6e7a-db4e-11e7-8627-0682d9894e02.png)
![image](https://user-images.githubusercontent.com/30419014/33714305-c07e3adc-db4e-11e7-89d7-88438f7e42a2.png)

shemogumbe commented on .eslintrc:
```diff
@@ -33,6 +33,11 @@
     "react"
   ],
   "rules": {
+    "import/no-absolute-path": 0,
```
Are these changes really relevant for purposes of this PR?

tosmak16 replied:
Yes , they are necessary.

shemogumbe commented:
This PR has merge conflicts on two fields, the changes in .eslintrc that cause merge conflicts are even unnecessary for the task at hand.

tosmak16 committed:
feature(add-cost-price):
Add cost price to product variant schema

Add Cost prict Input field to product variant form

[finishes #153470982
